pkg/operator/api/v1alpha1/gameserver_types.go

```diff
@@ -80,6 +80,8 @@ type GameServerStatus struct {
 	Health GameServerHealth `json:"health,omitempty"`
 	// State defines the state of the game server (Initializing, StandingBy, Active etc.)
 	State GameServerState `json:"state,omitempty"`
+	// The Previously known manually set state
+	PrevState GameServerState `json:"prevstate,omitempty"`
```

Comment:
Current server handling is done mostly in "anonymous" bulk. Servers can change some states without the controller having knowledge to the change. This tag simply allows better tracking of a server's change, which in turn allows for more precise metric emission.